src/components/index/DeFiChainEcoSystemSection.tsx

```diff
@@ -76,43 +49,40 @@ export function DeFiChainEcoSystemSection(): JSX.Element {
             </span>
           </span>
         </div>
-        <div className="no-scrollbar grid grid-flow-row grid-cols-1 gap-6 md:grid-flow-row md:grid-cols-2 lg:block lg:h-[639px] lg:place-self-end">
+        <div className="grid grid-flow-row grid-cols-1 gap-x-6 sm:gap-y-6 md:grid-flow-row md:grid-cols-2 lg:block lg:h-[639px] lg:place-self-end">
```

Comment:
check spacing on mobile..

Reply:
![image](https://user-images.githubusercontent.com/44501120/227161430-89631a0d-8f8b-410e-9da0-3de33a93eded.png)
are you referring to this?